Lib/test/datetimetester.py

```diff
@@ -5991,30 +5991,36 @@ class DateTimeSubclass(datetime):
 
         for klass in [datetime, DateTimeSubclass]:
             for args in [(1993, 8, 26, 22, 12, 55, 99999),
-                         (1993, 8, 26, 22, 12, 55, 99999)]:
+                         (1993, 8, 26, 22, 12, 55, 99999,
+                          FixedOffset(0, "UTC", -2))]:
```

Comment:
I think I would just either use `datetime.timezone.utc` or another `datetime.timezone` fixed offset zone, unless you had a specific reason to pick this one.

Reply:
There was no specific reason. :)

`datetime.timezone.utc` is now used instead.